pkg/components/state/pluggable_test.go

```diff
@@ -237,7 +237,7 @@ func TestComponentCalls(t *testing.T) {
 		stStore, cleanup, err := getStateStore(svc)
 		require.NoError(t, err)
 		defer cleanup()
-		err = stStore.Delete(&state.DeleteRequest{
+		err = stStore.Delete(context.TODO(), &state.DeleteRequest{
```

Comment:
As is the case also in components-contrib, we use `context.Background()` in tests as this is not a TODO for the future.

Reply:
Fixed.